### ISSUE TITLE
Would like to sort the all Tags with Dropdown menu

### ISSUE THREAD

**Issue by Hamsterun** [starik222/BooruDatasetTagManager#19]
^
Would like to define Dropdowns with a custom title and add Tags, in the "all" tag section, to them for easier navigation when looking for something.
As I need specific Tags in a specific order.
So I don't have to think about the alphabetical order.
Would be neat. Thanks.

**Comment by starik222** [comment]
I didn't quite understand what you want. Now the third column has "all tags" and "common tags", do you want a third state with its own set of tags loaded from a text file?

**Comment by Hamsterun** [comment]
![tempsnip](https://user-images.githubusercontent.com/130394471/230969839-d1b48061-22a7-45ab-9b1e-c7f4a5b0ce28.png)
Yeah sorry, maybe this helps?

And something i just tried doing is to shift select multiple tags of an image, copy them with ctrl+c and paste them into another image with ctrl+v, which didn't work but felt natural to do. Of course, I used the copy paste feature afterwards. In case you got time to implement this workflow at some point.

**Comment by starik222** [comment]
Sorry, but I still can't understand your idea.
Do you want to get different lists (sets) of tags? That is, you add a list, then from the list of all tags add the tags you need to this list, and then switch from "all tags" to the list you need?

> And something i just tried doing is to shift select multiple tags of an image, copy them with ctrl+c and paste them into another image with ctrl+v, which didn't work but felt natural to do. Of course, I used the copy paste feature afterwards. In case you got time to implement this workflow at some point.

Keyboard shortcuts not implemented yet. #16

**Comment by Hamsterun** [comment]
Yeah exactly. You can have multiple Dropdown lists within the "all tags", best if they would be all at the top. Then you can drag, or somehow else, the tags into a dropdown list to sort it into that das dropdown list. Though every new tag will be sorted as it does now.
And even if one or more dropdown lists are open, the other normal sorted all tags should still be listed below.

Does that make sense? Had only about 3 hours with your tool and just wished I could somehow sort the tags I am using in the order I want to use them. Of course, I dunno if it can be done somehow else.

**Comment by renwerx** [comment]
I'd like to second this request. I'm thinking something like a file browser folder structure where the root can contain single tags or expandable "folders" of tags. New tags are added to the root by default and the user can drag and drop into or out of a folder. 

The use case I'm thinking of would be to have for instance a couple of primary tags then an "expressions" folder, a "poses" folder, a "background" folder etc.. 

This would help remove clutter and if you can manually order the list it will also help to enforce a consistent structure as one can move from top to bottom in the list adding conceptually relevant tag options in a defined sequence (first "expression" then "pose" then "background"). Happy to mockup the idea if I'm not explaining succinctly.
